lib/src/components/NxSearchDropdown/__tests__/NxSearchDropdown.test.tsx

```diff
@@ -4,225 +4,203 @@
-        .toHaveProp('role', 'menu');
-  });
+    const dropdownMenu = quickRender({ searchText: 'asdf', error: 'foo' }).getAllByRole('alert')[0];
+    expect(dropdownMenu).toBeInTheDocument();
 
-  it('sets the alert role on the NxDropdownMenu when it is in loading, error, or empty states', function() {
-    expect(getShallow({ searchText: 'asdf', matches: [] }).find(NxDropdownMenu)).toHaveProp('role', 'alert');
-    expect(getShallow({ searchText: 'asdf', loading: true }).find(NxDropdownMenu)).toHaveProp('role', 'alert');
-    expect(getShallow({ searchText: 'asdf', error: 'foo' }).find(NxDropdownMenu)).toHaveProp('role', 'alert');
+    const errorAlert = within(dropdownMenu).getByRole('alert');
+    expect(errorAlert).toBeInTheDocument();
   });
 
-  it('sets an id on the NxDropdownMenu and references it in the search boxes aria-controls', function() {
-    const component = getMounted({ searchText: 'foo' }),
-        filterInput = component.find(NxFilterInput).find('input'),
-        dropdown = component.find('.nx-dropdown-menu');
+  it('sets an id on the dropdown and references it in the search boxe\'s aria-controls', function() {
+    const el = quickRender({ searchText: 'foo' }),
+        filterInput = el.getByRole('searchbox'),
+        dropdown = el.getByRole('alert');
 
-    expect(dropdown).toHaveProp('id');
-    expect(filterInput).toHaveProp('aria-controls', dropdown.prop('id'));
+    expect(dropdown).toHaveAttribute('id');
+    expect(filterInput).toHaveAttribute('aria-controls', dropdown.getAttribute('id'));
   });
 
   it('sets aria-haspopup on the filter input', function() {
-    const component = getMounted({ searchText: 'foo' }),
-        filterInput = component.find(NxFilterInput).find('input');
-
-    expect(filterInput).toHaveProp('aria-haspopup', 'menu');
-  });
+    const el = quickRender({ searchText: 'foo' }),
+        filterInput = el.getByRole('searchbox');
 
-  it('renders an NxLoadWrapper within the dropdown menu', function() {
-    expect(getShallow({ searchText: 'foo' }).find(NxDropdownMenu)).toContainMatchingElement(NxLoadWrapper);
+    expect(filterInput).toHaveAttribute('aria-haspopup', 'menu');
   });
 
-  it('passes the loading prop to the load wrapper', function() {
-    const loading = getShallow({ searchText: 'foo', loading: true }),
-        notLoading = getShallow({ searchText: 'foo' }),
-        explicitNotLoading = getShallow({ searchText: 'foo', loading: false });
+  it('shows loading text on the dropdown when loading prop is true', function() {
+    const el = quickRender({ searchText: 'foo', loading: true });
 
-    expect(loading.find(NxLoadWrapper)).toHaveProp('loading', true);
-    expect(notLoading.find(NxLoadWrapper)).toHaveProp('loading', undefined);
-    expect(explicitNotLoading.find(NxLoadWrapper)).toHaveProp('loading', false);
+    expect(el.getByRole('status')).toHaveTextContent('Loading…');
```

Comment:
Should probably assert that this `status` is within the `alert`

Reply:
> Should probably assert that this `status` is within the `alert`

Added here: 063165dfc3227116141829180b041706b4eae005